README.md

```diff
@@ -24,3 +24,11 @@ Data graciously provided by [IsThereAnyDeal](https://isthereanydeal.com).
 Note: I recommend Violentmonkey since it is Free, Open-Source, and light on system resources. Tampermonkey is good, but it is proprietary.
 
 2. **[Click here to install the userscript](https://raw.githubusercontent.com/kevinfiol/enhanced-gog/master/bin/enhanced-gog.user.js)**
+
+## Building
+
+If you want to build a custom version of the script from the sources:
+
+1. Install [Node.js](https://nodejs.org/)
+2. From the root directory, run `npm install` to install the dependent packages
+3. Next run `npm run build` to build the script. The compiled script will be in 'bin/enhanced-gog.user.js`
```

Comment:
I left this comment on the stale PR, but posting here as well:

Thanks for adding this. Can you change the section name to `Development`? And a few points:

* This project uses `pnpm` as the package manager. `npm` will work, but will be incompatible with the lockfile, so there is a chance, albeit small, that you can install bad dependencies. You can add to step 1 or make step 2: "Install pnpm: `npm install -g pnpm`.

* There is also a dev script available that will start a local dev server: `pnpm run dev` or `pnpm dev`

* "dependent packages" can be "dependencies"

* Use tildes where you mention `'bin/enhanced-gog.user.js'`